company/models.py

```diff
@@ -49,7 +251,9 @@ class Company(models.Model):
     )
     official_name = models.CharField(max_length=255, blank=True, null=True, verbose_name=_("Nazwa rejestrowa"))
     common_name = models.CharField(max_length=255, blank=True, verbose_name=_("Nazwa dla użytkownika"))
-
+    capital_group = models.ForeignKey(
+        CapitalGroup, null=True, on_delete=models.CASCADE, blank=True, verbose_name=_("Grupa kapitałowa")
```

Comment:
Ja się boje ustawiać pola na `models.CASCADE` zwłaszcza, gdy są one dość wysoko hierarchi. Bezpieczniej będzie ustawić to pole na `models. PROTECT`